.github/workflows/main.yml

```diff
@@ -29,5 +27,29 @@ jobs:
       - name: Espresso Checks
         uses: reactivecircus/android-emulator-runner@v2.13.0
         with:
-          api-level: 23
+          api-level: 16
```

Comment:
This change seems unrelated – why lower this number?

Reply:
Because I asked to in the previous PR :) . The Travis CI ran on 16 so we shouldn't lose that. Though ideally we'd run on a few versions maybe.

Reply:
I can make a matrix to run tests on multiple versions in parallel, which ones do you think we should be testing?